Ignore build-related files produced by pip install.

sb10 opened the pull request:
`pip install` produces `build/` and `hgi_vault.egg-info/` directories, which should be ignored so they don't clutter up `git status` output and don't accidentally get added to the repo.

Are there any other commands we run that produced build-related files that should also be ignored?

sb10 committed:
Ignore build-related files produced by pip install.

michael-grace commented:
might be a good idea to use github's `.gitignore` template for Python rather than recreating our own

sb10 commented:
> might be a good idea to use github's `.gitignore` template for Python rather than recreating our own

Agreed, but wasn't sure how to get it for an existing repo. There's https://github.com/github/gitignore/blob/main/Python.gitignore, but it seems like a lot of stuff. Safe to add?

michael-grace committed:
Lock MarkupSafe to 2.0.1

Fixes:
- As our version of Jinja2 doesn't lock a version of MarkupSafe, it now breaks because of major version changes. This locks the version of MarkupSafe

michael-grace commented:
yeah, that link looks good

sb10 committed:
Merge branch 'fix/markupsafe-version' of github.com:wtsi-hgi/hgi-vault into gitignore

sb10 committed:
Use githubs full python gitignore.
